server-embedded/pom.xml

```
@@ -156,7 +156,7 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>@{argLine} --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED -Dde.flapdoodle.os.override=Linux|X86_64|Ubuntu|Ubuntu_16_04 -Dchat.embeddedMongoDBVersion=4.2</argLine>
+          <argLine>@{argLine} --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED -Dde.flapdoodle.os.override=Linux|X86_64|Ubuntu|Ubuntu_22_10 -Dchat.embeddedMongoDBVersion=6.0</argLine>
```

```suggestion
          <argLine>@{argLine} --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED -Dde.flapdoodle.os.override=Linux|X86_64|Ubuntu|Ubuntu_20_04 -Dchat.embeddedMongoDBVersion=6.0.6</argLine>
```
the OS included in the ci and release docker images is 20.04